### DISCUSSION TITLE
Grid-item keeps reverting to original position after drag-end

### DISCUSSION THREAD

**Discussion by Namidas** [jbaysolutions/vue-grid-layout#704] [General]
Hi, I'm facing a strange issue, when grid-item drag ends, it goes back to it's original position.
I'm printing the layout, so I can see that it updates properly while dragging, but then just stays there, the element reverts as if a cancel drag of some sort has been activated, and the property "moved" for the item always remains false.

The really strange thing is that if copy/paste any given example, it works perfectly =\

Here's the template I'm using for the grid...

```
<grid-layout
        :col-num="12"
        :row-height="30"
        :is-draggable="true"
        :is-resizable="true"
        :is-bounded="true"
        :vertical-compact="false"
        :margin="[30, 30]"
        :use-css-transforms="true"
        v-model:layout="desktopLayout"
      >
      <my-widget
        v-for="widget in desktopWidgets"
        :key="widget.widget"
        v-bind="widget"
      />
</grid-layout>
```

and these are my grid items...

```
<grid-item
        :i="widgetProps.layout.i"
        :x="widgetProps.layout.x"
        :y="widgetProps.layout.y"
        :w="widgetProps.layout.w"
        :h="widgetProps.layout.h"
        drag-allow-from=".draggable-handle"
        drag-ignore-from=".no-drag"

        @resize="resizeEvent"
        @move="moveEvent"
        @resized="resizedEvent"
        @container-resized="containerResizedEvent"
        @moved="movedEvent"
        >
        <div class="my-widget" :data-name="widget">
            <component :is="widget" :widget="widget" :widgetProps="widgetProps" ref="compo"></component>
        </div>
    </grid-item>
```

Any thoughts on this?
Any idea where to start looking at ?

**Comment by Namidas** [comment]
Just in case anyone is facing this same issue, I, for some CMS/backend reasons was keeping the layout and each widget data repeated and separated. If you check my code, you'll see you have a layout and then you have item props which have some layout info.

For whatever reason that I won't explain, that seemed just right, problem is grid-item requires that any props/data/whatever is a proxy from whatever you're using for a model in grid-layout.

That's why it was rendering just right (propper data), but it wasn't reacting to anything else.

**In other words, be sure to feed grid-item with a proxy of whatever items you're feeding grid-layout with, otherwise it won't work.**
